util.py

```diff
@@ -61,26 +61,27 @@ def get_data_as_csv(self, file_path):
             TODO: return dict format to be standardized
         """
         new_count = 0
-        search = self.ds_config['search']
-        count = int(self.ds_config['count'])
-        max_count = count + const.CONFIG_FIELDS.COUNT_INCREMENT
+        search = self.ds_config[const.CONFIG_FIELDS.SEARCH]
+        count = int(self.ds_config[const.CONFIG_FIELDS.COUNT])
+        max_count = count + const.MAX_LIMIT
         response = get_wikipedia_streams_data()
-        field_names = const.FIELDS_NAME
+        field_names = const.FIELDS_NAMES
         with open(file_path, 'w') as outfile:
             writer = csv.DictWriter(outfile, fieldnames=field_names)
             idx = max_count
             if search != "":
                 for event in response.events():
                     change = json.loads(event.data)
-                    change = extract_predefined_dict(change, field_names)
```

Comment:
since we already defined field_names in const.py, no need to pass it as parameter, use it directly from there.